### PR TITLE
Reflect closed issues and issues with linked pull requests in task list article

### LINKED ISSUE #13031: Update about task list md to reflect closed PR
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

**about-task-list.md**

https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists

image
These icons are only available on open issues with task list. Pull Request with task list items has already been submitted are unable to create new issues with task list items.

image

image

### What changes are you suggesting?

Updated the about task list documentation to reflect that you can not create task list items issues from closed or issues you have started a pull request for.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by cadamini**
### Why:

Closes https://github.com/github/docs/issues/13031

### What's being changed:

* Simplified texts a bit
* Added a sentence about task not available in closed issues or issue with linked pull requests.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/issues/tracking-your-work-with-issues/about-task-lists.md [hunk 1]
```diff
@@ -51,9 +51,11 @@ If you add a task list to the body of an issue, the list has added functionality
 
 {% data reusables.repositories.task-list-markdown %}
 
+You cannot create task list items issues from closed issues or issues with linked pull requests.
```
Comment (cadamini) [inline content/issues/tracking-your-work-with-issues/about-task-lists.md:54]:
```suggestion
You cannot create task list items from closed issues or issues with linked pull requests.
```

**Comment by ramyaparimi** [comment]
@cadamini 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Karol12347 — COMMENTED** [review]
.. @

File: content/issues/tracking-your-work-with-issues/about-task-lists.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ topics:
 
 ## About task lists
 
-A task list is a set of tasks that each render on a separate line with a clickable checkbox. You can select or deselect the checkboxes to mark the tasks as complete or incomplete. 
+A task list is a set of tasks that each render on a separate line with a clickable checkbox. Select or deselect the checkboxes to mark the tasks as complete or incomplete.
```
Comment (mchammer01) [inline content/issues/tracking-your-work-with-issues/about-task-lists.md:29]:
```suggestion
A task list is a set of tasks that each render on a separate line with a clickable checkbox. You can select or deselect the checkboxes to mark the tasks as complete or incomplete. 
```

File: content/issues/tracking-your-work-with-issues/about-task-lists.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ If you add a task list to the body of an issue, the list has added functionality
...
 - If a task references another issue and someone closes that issue, the task's checkbox will automatically be marked as complete. 
-- If a task requires further tracking or discussion, you can convert the task to an issue by hovering over the task and clicking {% octicon "issue-opened" aria-label="The issue opened icon" %} in the upper-right corner of the task. To add more details before creating the issue, you can use keyboard shortcuts to open the new issue form. For more information, see "[Keyboard shortcuts](/github/getting-started-with-github/using-github/keyboard-shortcuts#issues-and-pull-requests)."
+- If a task requires further tracking or discussion, convert the task to an issue by hovering over the task and clicking {% octicon "issue-opened" aria-label="The issue opened icon" %} in the upper-right corner of the task. To add more details before creating the issue, you can use keyboard shortcuts to open the new issue form. For more information, see "[Keyboard shortcuts](/github/getting-started-with-github/using-github/keyboard-shortcuts#issues-and-pull-requests)."
```
Comment (mchammer01) [inline content/issues/tracking-your-work-with-issues/about-task-lists.md:43]:
```suggestion
- If a task requires further tracking or discussion, you can convert the task to an issue by hovering over the task and clicking {% octicon "issue-opened" aria-label="The issue opened icon" %} in the upper-right corner of the task. To add more details before creating the issue, you can use keyboard shortcuts to open the new issue form. For more information, see "[Keyboard shortcuts](/github/getting-started-with-github/using-github/keyboard-shortcuts#issues-and-pull-requests)."
```

File: content/issues/tracking-your-work-with-issues/about-task-lists.md [hunk 4]
```diff
@@ -51,9 +51,11 @@ If you add a task list to the body of an issue, the list has added functionality
 
 {% data reusables.repositories.task-list-markdown %}
 
+You cannot create task list items within closed issues or issues with linked pull requests.
+
 ## Reordering tasks
 
-You can reorder the items in a task list by clicking to the left of a task's checkbox, dragging the task to a new location, and dropping the task. You can reorder tasks across different lists in the same comment, but you can not reorder tasks across different comments.
+Reorder the items in a task list by clicking to the left of a task's checkbox, dragging the task to a new location, and dropping the task. You can reorder tasks across different lists in the same comment, but you can not reorder tasks across different comments.
```
Comment (mchammer01) [inline content/issues/tracking-your-work-with-issues/about-task-lists.md:58]:
```suggestion
You can reorder the items in a task list by clicking to the left of a task's checkbox, dragging the task to a new location, and dropping the task. You can reorder tasks across different lists in the same comment, but you can not reorder tasks across different comments.
```

File: content/issues/tracking-your-work-with-issues/about-task-lists.md [hunk 5]
```diff
@@ -51,9 +51,11 @@ If you add a task list to the body of an issue, the list has added functionality
 
 {% data reusables.repositories.task-list-markdown %}
 
+You cannot create task list items within closed issues or issues with linked pull requests.
```
Comment (mchammer01) [inline content/issues/tracking-your-work-with-issues/about-task-lists.md:54]: I like your suggestion here, just making it into a tip so it's more prominent 🙂 
```suggestion
{% tip % }

**Tip:** You cannot create task list items within closed issues or issues with linked pull requests.

{% endtip %}
```

**Review by mchammer01 — COMMENTED** [review]
@cadamini 👋🏻 - thanks for improving our docs ✨ 
I've reverted your text simplifications as these are not in accordance with our content models at GitHub.
I'll apply my suggestions and then this should be ready for merging.

**Review by mchammer01 — APPROVED** [review]
LGTM ✨

**Pull request merged**